types/node/index.d.ts

```diff
@@ -5254,6 +5255,21 @@ declare module "util" {
         __promisify__: TCustom;
     }
 
+    export function callbackify(fn: () => Promise<void>): (callback: (err: NodeJS.ErrnoException) => void) => void;
```

Comment:
I think it should be `NodeJS.ErrnoException | null`.

Reply:
🤔 After checking the definition of NodeJS.ErrnoException, I'm convinced that it should be the case. However, it seems like no one use `NodeJS.ErrnoException | null` in all other callback functions.

We can either correct all of them, or convert `ErrnoException` to a type which includes `null` as an alternative.

Reply:
I think it would be better to add `| null` everywhere, because people may be using `NodeJS.ErrnoException` as a type already and expecting it to be an interface type.

Reply:
I agree with @andy-ms. I expect node types were created before strictNullChecks was in place.
But I think such a change should be in a new PR. Maybe I will create one once CI for Node is stable again...